src/snyk/snykCode/views/suggestion/codeSuggestionWebviewScript.ts

```diff
@@ -2,7 +2,7 @@
 /* eslint-disable @typescript-eslint/no-non-null-assertion */
 /* eslint-disable @typescript-eslint/no-explicit-any */
 /* eslint-disable @typescript-eslint/no-unsafe-member-access */
-/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+
```

Comment:
I have decided to enable this rule with the idea that little by little we will add the missing types. I'm happy to revert it if the team finds it so.